src/yunohost/app.py

```diff
@@ -2105,3 +2106,10 @@ def random_password(length=8):
 
     char_set = string.ascii_uppercase + string.digits + string.ascii_lowercase
     return ''.join([random.SystemRandom().choice(char_set) for x in range(length)])
+
+
+def normalize_url_path(url_path):
```

Comment:
probably nitpicking, but this helper could be in a file within the https://github.com/YunoHost/yunohost/tree/unstable/src/yunohost/utils folder ? (maybe out of scope of this PR)

Reply:
Probably. It would be ever greater in moulinette utils I think but that's mostly a "quick fix" PR to unblock the app group situation. I don't mind if we do those kind of modifications (as long as it don't block this PR for the next release) but I'm not super available for that now :x

Reply:
Like this one ? :wink: 

https://github.com/YunoHost/yunohost/blob/3571747718ea47e7a2d522adc8d5b2c03c64d261/data/helpers.d/network#L1-L23

Reply:
That doesn't great to call that from python :x

Reply:
Ah nvm I was thinking @opi wanted to make this available for apps

Reply:
no, but not scoped in app.py ; But let's merge it now, and we'll cleanup later.